packages/api/src/models/db-dev.ts

```diff
@@ -64,6 +64,68 @@ const createTokenTable = async (ddb: AWS.DynamoDB): Promise<void> => {
     await ddb.createTable(params).promise();
   }
 };
+// Creates the rate limting tracaking table
+const createRateLimitTrackingTable = async (ddb: AWS.DynamoDB): Promise<void> => {
+  if (!docTableNames.rateLimitingTracking) return;
+  const doesTableExist = await tableExists(docTableNames.rateLimitingTracking, ddb);
+  if (!doesTableExist) {
+    const params: AWS.DynamoDB.CreateTableInput = {
+      AttributeDefinitions: [
+        {
+          AttributeName: "cxId_operation",
+          AttributeType: "S",
+        },
+        {
+          AttributeName: "window_timestamp",
+          AttributeType: "S",
+        },
+      ],
+      KeySchema: [
+        {
+          AttributeName: "cxId_operation",
+          KeyType: "HASH",
+        },
+        {
+          AttributeName: "window_timestamp",
+          KeyType: "RANGE",
+        },
+      ],
+      ProvisionedThroughput: {
+        ReadCapacityUnits: 1,
+        WriteCapacityUnits: 1,
+      },
+      TableName: docTableNames.rateLimitingTracking,
+    };
+    await ddb.createTable(params).promise();
+  }
+};
+// Creates the rate limiting settings table
+const creatSettingsTable = async (ddb: AWS.DynamoDB): Promise<void> => {
```

Comment:
I get storing the rateLimitTracking in dynamo but why not store the settings in postgresql?

Reply:
Is it more efficient to store this in dynamodb since we are querying for it on every request?

Reply:
Yea exactly -- the settings lookup is exactly a key/value store (i.e. there's an exact setting value for a give key) so DDB is much more efficient than PG and we don't want this to be hitting our OSS db on every request occupying a connection.